src/UICrafter.Client/Components/AppViewDesigner.razor

```diff
@@ -0,0 +1,87 @@
+@using static UICrafter.Client.Pages.UICrafterPage
+
+<MudContainer MaxWidth="MaxWidth.Small">
+    <MudDropContainer T="UIComponent" @ref="DropContainer" Items="UIComponents" ItemsSelector="@((item,dropzone) => item.DropZoneID == dropzone)" ItemDropped="ComponentUpdated" Class="d-flex flex-wrap flex-grow-1">
+        <ChildContent>
+            <MudDropZone T="UIComponent" Identifier="Drop Zone 1" AllowReorder="true" Class="rounded mud-background-gray pa-2 flex-grow-1" Style="height: 45rem; overflow: auto;">
+            </MudDropZone>
+        </ChildContent>
+        <ItemRenderer>
+            @if (context is UIButton button)
+            {
+                <MudButton Class="mb-2" Variant="Variant.Filled" @onfocus="()=>SelectComponent(button)">@button.Label</MudButton>
```

Comment:
I am unsure if it is a good idea but Buttons can be disabled with a property making it so it is grayed out and wont call OnClick. You can try and see if you like it.

Reply:
Tror begge er fint, den har bare også lav oppacity når den er disabled lige så vidt jeg husker så tror jeg beholder det sådan her siden det ikk gør noget (tænker jeg ? :P)